apps/bare-expo/scripts/start-ios-e2e-test.ts

```diff
@@ -21,6 +21,11 @@ enum StartMode {
   try {
     const startMode = getStartMode();
 
+    if (startMode === StartMode.BUILD) {
+      TARGET_DEVICE = 'iPhone 15';
+      TARGET_DEVICE_IOS_VERSION = 17;
```

Comment:
could we have any better heuristic to get a proper device for the build? that would reduce the maintenance cost. i believe we have some heuristic inside `npx expo run:ios`